supporting_scripts/course-scripts/hyperion-benchmark-workflow/utils.py

```
@@ -0,0 +1,94 @@
+    :param str password: The password for authentication.
+    :param requests.Session session: The session object to use for the request.
+    :return: The response object from the authentication request.
+    :rtype: requests.Response
+    :raises Exception: If authentication fails (status code other than 200).
+    """
+    url: str = f"{SERVER_URL}/core/public/authenticate"
+    headers: Dict[str, str] = {
+        "Content-Type": "application/json"
+    }
+
+    payload: Dict[str, Any] = {
```

Ruff (0.14.14)

[error] Undefined name `Any`

(F821)